tests/features/api/places.feature

```diff
@@ -0,0 +1,16 @@
+Feature: Places by osm_type and osm_id Tests
+    Simple tests for internal server errors and response format.
+    These tests should pass on any Nominatim installation.
```

Comment:
Better to remove this comment. The tests as written require a planet installation.